contracts/contracts/strategies/UniswapStrategy.sol

```diff
@@ -0,0 +1,256 @@
+pragma solidity 0.5.11;
+pragma experimental ABIEncoderV2;
+
+// can't just import router - it wants >=0.6.2;
+/* import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol"; */
```

Comment:
We have an existing interface in interfaces/uniswap/, we can just add the additional methods to that and import it